src/brasil/gov/portal/upgrades/v10600/handler.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from Products.CMFCore.utils import getToolByName
+from brasil.gov.portal.config import PROJECTNAME
+
+import logging
+
+logger = logging.getLogger(PROJECTNAME)
+
+
+def install_product(context):
+    """Atualiza perfil para versao 10600"""
+    portal = getToolByName(context, 'portal_url').getPortalObject()
+    qi = getToolByName(portal, 'portal_quickinstaller')
```

Comment:
these 2 lines can be replaced by ``api.portal.get_tol('portal_url')`` using plone.api.